pkgs/development/python-modules/py-deprecate/default.nix

```diff
@@ -0,0 +1,37 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pytestCheckHook
+, scikit-learn
+, pytorch
+}:
+
+let
+  pname = "pyDeprecate";
+  version = "0.3.2";
+in
+buildPythonPackage {
+  inherit pname version;
+
+  src = fetchFromGitHub {
+    owner = "Borda";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-84RmQvLxwtLPQk3hX7Q6eeJeejhrO3t+mc95W1E85Fg=";
+  };
+
+  checkInputs = [
+    pytestCheckHook
+    scikit-learn
+  ];
+
+  meta = with lib; {
+    description = "A python module for marking deprecated functions or classes and re-routing to the new successors' instance. Used by torchmetrics";
+    homepage = "https://borda.github.io/pyDeprecate/";
+    license = licenses.asl20;
+    maintainers = with maintainers; [
```

Comment:
Please put maintainers into one line.

Reply:
> Please put maintainers into one line.

That'd spoil the git-blame next time someone extends the list

Reply:
Yes but that is more an issue for dependencies which easily diverge between branches. Maintainers shouldn't to much but how you want.

Reply:
I guess I'd rather keep it as it is 🤔